index.html

```diff
@@ -0,0 +1,175 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link
+      href="https://fonts.googleapis.com/icon?family=Material+Icons"
+      rel="stylesheet" />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css" />
+    <link rel="stylesheet" href="./style.css" />
+    <title>Shaheryar Capstone</title>
+    <link rel="stylesheet" href="style.css">
+  </head>
+  <body>
+    <div class="bgImg">
+      <!-- Header and Nav section -->
+      <header>
+        <div class="socialWrapper">
+          <div class="social">
+            <a href="#"><i class="fa fa-facebook"></i></a>
+            <a href="#"><i class="fa fa-twitter"></i></a>
+            <a href="#">English</a>
+            <a href="#">My Page</a>
+            <a href="#">Logout</a>
+          </div>
+        </div>
+
+        <nav class="nav-bar">
+          <div class="wrap-hamburger">
+            <button class="hamburger">
+              <i class="menuIcon material-icons">menu</i>
+              <i class="closeIcon material-icons">close</i>
+            </button>
+          </div>
+          <div class="logoListWrapper">
+            <div class="logo">
+              <a href="#">
+                <img
+                  src="img/logo.jpeg" height="60px"
+                  alt="logo"
+              /></a>
+            </div>
+            <ul class="nav-list">
+              <li class="navItem"><a href="/index.html">Home</a></li>
+              <li class="navItem"><a href="/about.html">About</a></li>
+              <li class="navItem"><a href="#programs">Program</a></li>
+              <li class="navItem" id="join"><a href="#join">Join</a></li>
+              <li class="navItem"><a href="#speakers">Speakers</a></li>
+              <li class="navItem"><a href="#sponser">Sponser</a></li>
+              <li class="navItem box"><a href="#">PE Campaign</a></li>
+            </ul>
+          </div>
+        </nav>
+      </header>
+
+      <!-- Main section and Headline Start -->
+
+      <section class="main">
+        <div class="headline">
+          <h4>"Hello! Sharing world"</h4>
+          <h1>PAKISTAN ECOMERCE <br> SUMMIT 2022</h1>
+        </div>
+        <p class="message">
+          A fun festival where people from more than 80 countries who believe in
+          the value of sharing and openness and create positive change gather
+          and share ideas will be held in Korea in October.
+        </p>
+        <div class="location">
+          <h1 class="date">2022.10.27 (Fri) ~ 29 (Sun)</h1>
+          <h5 class="locate">
+            @ Islamabad club pakistan.
+          </h5>
+        </div>
+      </section>
```

Comment:
- [ ] I have just realized that this section on the desktop does not have the same margins/ padding bottom as it is in the Behance design. Kindly consider adding more padding/ margin-bottom to make it look more of the Behance design.

**This is how I am seeing your design:**

![image](https://user-images.githubusercontent.com/86318284/198352773-1ae035bd-d768-4fa7-aa65-81bca43d03e4.png)

**This is the desired design:**

![image](https://user-images.githubusercontent.com/86318284/198352860-3b52a541-9cbe-466d-aa32-3e0097a9c4c3.png)


- [ ] On mobile, the lower section of the landing section is overlapped and hidden by the section below it and it cannot be visible altogether. Kindly look into why this is the case.

**This is how I am seeing the section:**

![image](https://user-images.githubusercontent.com/86318284/198353405-e129db7d-d8b5-44a8-93bd-262b320a00f8.png)

**This is the desired design:**

![image](https://user-images.githubusercontent.com/86318284/198353523-da12314a-3466-4ad4-ac4f-5f119e6656b8.png)